build-coordinator/src/main/java/org/jboss/pnc/core/builder/BuildCoordinator.java

```diff
@@ -378,4 +375,8 @@ private boolean isBuildAlreadySubmitted(BuildTask buildTask) {
     Event<BuildSetStatusChangedEvent> getBuildSetStatusChangedEventNotifier() {
         return buildSetStatusChangedEventNotifier;
     }
+
+    public void shutdownCoordinator(){
```

Comment:
I would be good to invoke it somewhere... Maybe a `@Predestroy` method or something similar?